### ISSUE TITLE
working example

### ISSUE THREAD

**Issue by cotestatnt** [atanisoft/esp_lcd_touch_xpt2046#2]
Hello @atanisoft  and thank you for sharing your library.

Could you provide some working example that makes use of the LVGL framework with ESP-ISD v5.x.x?
It would be greatly appreciated.

Thanks!

**Comment by atanisoft** [comment]
@cotestatnt  https://github.com/espressif/esp-idf/tree/master/examples/peripherals/lcd/spi_lcd_touch is probably the best example for usage at this time. Being a touch only component it is problematic to add an LVGL example as there is no specific display component in use here.

**Comment by limpens** [comment]
Hi @atanisoft , thanks for the component! I was following the other issue #3 , and just did an attempt to get the driver working with an ILI9341 based board.

I [uploaded](https://github.com/limpens/esp32-2432S028) a really basic example, maybe this helps others.

**Comment by aditpodungge19** [comment]
@atanisoft Thank you for sharing this dependency. I'm still learning to use esp-idf, are there any examples that I can use? Thank you

**Comment by limpens** [comment]
I am using this touch driver in a simple example, maybe [this](https://github.com/limpens/esp32-2432S028R) helps?

**Comment by aditpodungge19** [comment]
@limpens thank you, but I will try to learn it, because I use ili9488. so there will be difficulties.

**Comment by aditpodungge19** [comment]
Hi @limpens , thank you for the example. However, I'm still confused because I'm using ili 9488

**Comment by aditpodungge19** [comment]
> @aditpodungge19 For your ILI9488 you should be able to use https://github.com/atanisoft/esp_lcd_ili9488 via IDF component manager without issues, there are also a couple examples available using LVGL.

Yes, I used that dependency but I couldn't integrate xpt2046. Thank you for the dependency.
